format/index.md

```diff
@@ -550,8 +555,14 @@ A server **MUST** respond to a successful request to fetch an individual
 resource with a *resource object* or `null` provided as the response
 document's primary data.
 
-> Note: `null` is only an appropriate response for fetching a to-one
-related resource URL to indicate the absence of a resource in the relationship.
+`null` is only an appropriate response when the requested URL is one that might
+correspond to a single resource, but doesn't currently.
+
+> Note: To clarify, consider a to-one related resource URL. This URL can
+correspond to a single resource (the related one), but it can also correspond to
+no resources when the relationship is empty. Therefore, a request to fetch a
+to-one related resource URL would respond with `null` when there is no resource
+in the relationship and with a single resource object otherwise.
```

Comment:
I'm working to shorten the note now, which is definitely an improvement, but I do think a standalone note with a bit more text would be helpful for people who aren't super familiar with the spec